microsoft-365/compliance/tls-1-2-in-office-365-gcc.md

```diff
@@ -1,28 +1,26 @@
 ---
 title: Deprecating TLS 1.0 and 1.1 in Office 365 GCC High and DoD
 description: Discusses how Microsoft is moving the date forward to discontinue support for TLS 1.1 and 1.0 in GCC High and DoD environments in Office 365 and preparing to use TLS 1.2.
-author: simonxjx
-manager: dcscontentpm
+author: workshay
+manager: laurawi
 localization_priority: Normal
 search.appverid: 
 - MET150
 audience: ITPro
-ms.service: O365-seccomp
+ms.collection: M365-security-compliance
+ms.service: information-protection
 ms.topic: article
-ms.author: v-six
-ms.reviewer: lobrion
+ms.reviewer: krowley
+ms.auther: shmehta
 appliesto:
 - Office 365 Business
 ---
 
 # Deprecating TLS 1.0 and 1.1 in Office 365 GCC High and DoD
 
-> [!IMPORTANT]
-> The world is in the middle of a pandemic, and we at Microsoft are aware of the impact on our customers and partners. To lighten the burden on our commercial customers, we have temporarily halted any deprecation enforcement of TLS 1.0 and 1.1. An update will be sent on a revised timeline after the current crisis stabilizes. (This article is revised to reflect the change.)
-
 ## Summary
 
-In order to comply with the latest compliance standards for the Federal Risk and Authorization Management Program (FedRAMP), we are moving to deprecate Transport Layer Security (TLS) versions 1.1 and 1.0 in Microsoft Office 365 for GCC High and DoD environments. This change was previously announced through Microsoft Support in [Preparing for the mandatory use of TLS 1.2 in Office 365](https://support.microsoft.com/help/4057306/preparing-for-tls-1-2-in-office-365).
```

Comment:
Should "we are deprecating Transport Layer Security (TLS) versions 1.1 and 1.0 ..." really be "we have deprecated"?? 
Response: I don't think we've promised customers an end date on deprecation though. It's essentially a from this date forward, you may experience data loss, your connections may stop working, tl;dr things might break because of this and we're not going to help you outside of tell you to upgrade to TLS 1.2. Also, Skype has no plans to deprecate, they will just retire all up.
Comment: we have definitions for deprecated v. no longer supported v. will cause data loss if you continue to use it v. end of life v. known to cause a security risk. until we identify the accurate wording and the final date when the last product completes, leave deprecating as is. I will escalate with the term team.